msteams-platform/apps-in-teams-meetings/in-meeting-notification-for-meeting-stage.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Build in-meeting notification for Teams meeting
+author: v-sdhakshina
+description: In this article, learn how to build in-meeting notification for Microsoft Teams meeting.
+ms.topic: conceptual
+ms.author: v-sdhakshina
+ms.localizationpriority: medium
+---
+
+# Build in-meeting notification for Teams meeting
+
+The in-meeting notification is used to engage participants during the meeting and collect information or feedback during the meeting. Use an [in-meeting notification payload](advanced-meeting-apis.md#send-an-in-meeting-notification) to trigger an in-meeting notification. As part of the notification request payload, include the URL where the content to be shown is hosted.
+
+In-meeting notification must not use task module. Task module isn't invoked in a meeting chat. An external resource URL is used to display in-meeting notification. You can use the `submitTask` method to submit data in a meeting chat.
```

Comment:
This paragraph doesn't quite make sense to me... not sure what the topic is that relates these 4 sentences.

Reply:
@fallothakor Do you know what this paragraph is trying to convey?